vespabase/src/vespa-configserver.service.in

```diff
@@ -1,6 +1,7 @@
 # Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 [Unit]
 Description=Vertical Search Platform Config Server
+After=network.target
```

Comment:
This should be 

```
After=network-online.target
Wants=network-online.target
```

according to https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/.

However it depends on having a correct wait serving running, and if not, it would probably wait indefinitely.